### PR TITLE
upgrade to 8.3

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

**Comment by davidanthoff** [comment]
Hm, tests fail, seems we need to adjust something else?

**Comment by SimonDanisch** [comment]
Thiss passes tests now! Would be nice to merge & tag!

**Comment by SimonDanisch** [comment]
Bump

File: Project.toml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 [compat]
-JSON = "0.19, 0.20, 0.21"
 FilePaths = "0.8"
+JSON = "0.19, 0.20, 0.21"
 URIParser = "0.4"
 julia = "1.3"
 
+[extras]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
```
[the remark this answers is not in the document]
Comment (SimonDanisch) [reply Project.toml:22]: Oh woops, I guess this must have happened when loading the env in Atom

**Comment by davidanthoff** [comment]
@SimonDanisch could change this PR so that it doesn't have any changes for `Project.toml`, and then I can merge this?

**Comment by SimonDanisch** [comment]
Sorry, now I was out of it, since I started working around the fact that this PR didn't get merged :D 
Sure make any change you want ;)

**Comment by SimonDanisch** [comment]
>Oh, and when I tag this, I should probably tag it as a new major version, right?

Yes, definitely!

**Pull request closed without merging**